src/Stripe.net/Services/PaymentIntents/PaymentIntentVerifyMicrodepositsOptions.cs

```diff
@@ -0,0 +1,16 @@
+// File generated from our OpenAPI spec
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class PaymentIntentVerifyMicrodepositsOptions : BaseOptions
+    {
+        /// <summary>
+        /// Two positive integers, in <em>cents</em>, equal to the values of the microdeposits sent
+        /// to the bank account.
+        /// </summary>
+        [JsonProperty("amounts")]
+        public List<long?> Amounts { get; set; }
```

Comment:
Do you know why it's a `List<long?>` here? I would have expected it to match the old BankAccount verification flow: https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/BankAccounts/BankAccountVerifyOptions.cs#L14